swift/schema.py

```diff
@@ -921,6 +921,7 @@ class StmtCondition(AstNode):
 
 class BraceStmt(Stmt):
     elements: list[AstNode] | child
+    variables: list[VarDecl] | child | doc("variable declared in the scope of this brace statement")
```

Comment:
variables should probably precede elements, so that they are printed first in the AST